sklearn_bench/dbscan.py

```diff
@@ -40,7 +40,10 @@ def main():
     labels = dbscan.labels_
 
     params.n_clusters = len(set(labels)) - (1 if -1 in labels else 0)
-    acc = davies_bouldin_score(X, labels)
+    try:
+        acc = davies_bouldin_score(X, labels)
+    except Exception:
```

Comment:
It would be better to have all scoring functions in the same place (bench.py)
And again, such handling is a bad practice